pkg/azurefile/azurefile.go

```diff
@@ -197,7 +195,10 @@ var (
 
 	retriableErrors = []string{accountNotProvisioned, tooManyRequests, shareBeingDeleted, clientThrottled}
 
+	// defaultAzcopyCopyOptions used in volume cloning and set --check-length to false if volume is in changing state
 	defaultAzcopyCopyOptions = []string{"--recursive", "--check-length=false"}
+	// snapshotAzcopyCopyOptions used in smb snapshot restore and set --check-length to true
```

Comment:
pls explain why `"--check-length=true"` is set in mount options for snapshot and not set in volume cloning

Reply:
done